src/screens/TaskDetailScreen.tsx

```diff
@@ -63,12 +99,24 @@ const TaskDetailScreen: React.FC<TaskDetailScreenProps> = ({ navigation, route }
   // Local state to manage the task
   const [task, setTask] = useState<Task>(initialTask);
 
+  // Animation values
+  const fadeAnim = useState(new Animated.Value(0))[0];
+
+  // Start fade-in animation when component mounts
+  React.useEffect(() => {
```

Comment:
Syncing external updates
If initialTask ever changes (for example, if the parent re‑renders with a new task), your local task state won’t update. To cover that, consider:

ts
Copy
Edit
React.useEffect(() => {
  setTask(initialTask);
}, [initialTask]);